connection.go

```diff
@@ -337,6 +334,83 @@ func (c *conn) executeStatement(ctx context.Context, query string, args []driver
 	return resp, err
 }
 
+func namedValuesToTSparkParams(args []driver.NamedValue) []*cli_service.TSparkParameter {
+	var ts []string = []string{"STRING", "DOUBLE", "BOOLEAN"}
```

Comment:
Should all number values be "DOUBLE"? What about INT, or DECIMAL?

Reply:
At the moment TSparkParameterValue only support bool, double and string values.  I'm waiting for clarification on the question of losing precision when converted to double.  Also waiting for clarification on whether TSparkParameter.Type should correspond to the type in TSparkParameterValue or if it is used for converting the value.

Reply:
From the code I saw in DBR, `TSparkParameter.Type` should correspond to the data types listed [here](https://docs.databricks.com/en/sql/language-manual/sql-ref-datatypes.html). In DBR, there will be an attempt to cast these to the correct type. Floats can even be represented as a string so we could do this (pseudocode):
```
TSparkParameter{Type: "FLOAT", Value: TSparkParameter{stringValue: "1234.125"}, ...}
```
Or 
```
TSparkParameter{Type: "DECIMAL(6, 2)", Value: TSparkParameter{stringValue: "1234.12"}, ...}
```
Or
```
TSparkParameter{Type: "BOOLEAN", Value: TSparkParameter{stringValue: "false"}, ...}
```
I think "ARRAY", "MAP", "STRUCT", and "BINARY" are not supported here however